docs/manual/docs/api/rdf-dcat.md

```diff
@@ -1,30 +1,42 @@
-
+The catalogue has the capability to convert ISO to DCAT format in various API endpoint. 
 
 ## Supported DCAT profiles
 
-A base conversion is provided with complementary extensions for various profiles of DCAT developed in Europe:
+A base conversion is provided with complementary extensions for various profiles of DCAT:
+
+| Profile                     | Version | Description                                                                                                         | URL  | Conversion from* |
+|-----------------------------|---------|---------------------------------------------------------------------------------------------------------------------|------|------------------|
+| W3C DCAT                    | 3       | Default W3C standard                                                                                                | https://www.w3.org/TR/vocab-dcat-3/  | ISO19115-3       |
+| European DCAT-AP            | 3.0.0 | DCAT profile for sharing information about Catalogues containing Datasets and Data Services descriptions in Europe  | https://semiceu.github.io/DCAT-AP/releases/3.0.0/  | ISO19115-3       |
+| European DCAT-AP-Mobility   | 1.0.1   | mobilityDCAT-AP is a mobility-related extension of the DCAT-AP                                                      | https://mobilitydcat-ap.github.io/mobilityDCAT-AP/releases/  | ISO19115-3       |
+| European DCAT-AP-HVD        | 2.2.0   | DCAT-AP for a dataset that is subject to the requirements imposed by the High-Value Dataset implementing regulation | https://semiceu.github.io/DCAT-AP/releases/2.2.0-hvd/  | ISO19115-3       |
+| European GeoDCAT-AP         | 3.0.0   |                                                                                                                     | https://semiceu.github.io/GeoDCAT-AP/releases/3.0.0/  | ISO19115-3       |
+| European GeoDCAT-AP (SEMIC) | 3.0.0   | [XSLT conversion maintained by SEMIC](https://github.com/SEMICeu/iso-19139-to-dcat-ap/blob/main/iso-19139-to-dcat-ap.xsl)                                                                             | https://semiceu.github.io/GeoDCAT-AP/releases/3.0.0/  | ISO19139         |
+
 
-* a default DCAT export following W3C standard https://www.w3.org/TR/vocab-dcat-3/
-* an extension for the European DCAT-AP https://semiceu.github.io/DCAT-AP/releases/3.0.0/
-  * an extension for the European GeoDCAT-AP https://semiceu.github.io/GeoDCAT-AP/releases/
-  * an extension for the European DCAT-AP-Mobility https://mobilitydcat-ap.github.io/mobilityDCAT-AP/releases/1.0.0/index.html
-  * an extension for the European DCAT-AP-HVD https://semiceu.github.io/DCAT-AP/releases/2.2.0-hvd/
+\*: 
+* The mapping is done from ISO19115-3 to DCAT* except for the SEMIC conversion which converts ISO19139 to GeoDCAT-AP. 
```

Comment:
It seems some wrong formatting with the `*` tags.

Reply:
It seems is not a formatting issue, the `*` is used as a reference to the table. I didn't understand that initially.